### PR TITLE
pd-configuration: add enable-joint-consensus

### PR DESCRIPTION
**Pull request by BusyJay**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add description for `enable-joint-consensus`.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by HunDunDM — APPROVED** [review]

File: pd-configuration-file.md [hunk 1]
```diff
@@ -303,6 +303,11 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 控制 TiKV 每分钟最多允许做 add peer 相关操作的次数。
 + 默认：15
 
+### `enable-joint-consensus`
```
Comment (TomShawn) [inline pd-configuration-file.md:306]:
```suggestion
### `enable-joint-consensus` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -303,6 +303,11 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
+
```
Comment (TomShawn) [inline pd-configuration-file.md:307]:
```suggestion

> **警告：**
>
> 目前 Joint Consensus 为实验特性，不推荐在生产环境中使用该特性。

```

File: pd-configuration-file.md [hunk 3]
```diff
@@ -303,6 +303,11 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
++ 是否使用 Joint Consensus 进行副本调度。关闭时，将采用一次调度一个副本的方式进行调度。本特性目前为实验特性。
```
Comment (TomShawn) [inline pd-configuration-file.md:308]:
```suggestion
+ 是否使用 Joint Consensus 进行副本调度。关闭该特性时，PD 将采用一次调度一个副本的方式进行调度。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: pd-configuration-file.md [hunk 4]
```diff
@@ -309,6 +309,15 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 + 控制 TiKV 每分钟最多允许做 add peer 相关操作的次数。
 + 默认：15
 
+### `enable-joint-consensus` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
+
+> **警告：**
+>
+> 目前 Joint Consensus 为实验特性，不推荐在生产环境中使用该特性。
```
Comment (scsldb) [inline pd-configuration-file.md:316]: 实验特性，不推荐在生产环境中使用。

**Pull request merged**